lib/pharos/phases/join_node.rb

```diff
@@ -22,7 +22,9 @@ def call
           join_command << '--cri-socket /var/run/crio/crio.sock'
         end
         join_command << "--node-name #{@host.hostname}"
-        join_command << "--ignore-preflight-errors DirAvailable--etc-kubernetes-manifests"
+        # CRI related pre-flight checks ignored as they cause false negative:
+        # https://github.com/kubernetes/kubeadm/issues/814
+        join_command << "--ignore-preflight-errors DirAvailable--etc-kubernetes-manifests,cri"
```

Comment:
Confirm that this broken in kubeadm 1.10.2 per https://github.com/kubernetes/kubernetes/pull/63297, but it looks like it's fixed in 1.10.3:

* https://github.com/kubernetes/kubernetes/blob/v1.10.2/cmd/kubeadm/app/preflight/checks.go#L107
* https://github.com/kubernetes/kubernetes/blob/v1.10.3/cmd/kubeadm/app/preflight/checks.go#L107

Reply:
so, now that kube 1.10.3 is merged to master, I'll merge/rebase and remove the cri pre-flight check ignore